### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by ThomasBarrial**

### CODE REVIEW

**Comment by ThomasBarrial** [comment]
Don't merge before redux and layout2 !!!!

**Review by MatthiasWanner — COMMENTED** [review]
Is it normal ?
Capture d’écran 2021-06-27 à 07 58 27

Capture d’écran 2021-06-27 à 07 59 05

File: src/views/HomePage.tsx [hunk 1]
```diff
@@ -69,40 +69,40 @@ function HomePage(): JSX.Element {
...
-      <div className="text-white font-roboto container">
-        <h1 className="text-8xl font-bold">aevisio</h1>
-        <h2 className="text-2xl">Expert Comptable.Audit.Conseil</h2>
+      <div className="text-white font-roboto flex items-center  flex-col">
+        <h1 className="sm:text-9xl text-8xl font-bold">aevisio</h1>
+        <h2 className="sm:text-2xl text-2xl">Expert Comptable.Audit.Conseil</h2>
       </div>
       <form
-        className="w-5/12 mt-10 flex flex-col text-white font-roboto text-xl"
+        className="w-12/12 sm:w-full mt-5 flex items-center flex-col text-white font-roboto text-xl sm:text-2xl"
```
Comment (MatthiasWanner) [inline src/views/HomePage.tsx:85]: w-12/12 & sm:w-full are egal.
In desktop mode, login form take full screen centered

File: src/components/Modal.tsx [hunk 2]
```diff
@@ -21,20 +21,20 @@ function Modal({ title, buttons, children }: Iprops): JSX.Element {
 
   return (
     <Dialog open={isOpen} onClose={handleClose} className="flex justify-center items-center fixed inset-0">
-      <Dialog.Overlay className="fixed inset-0 z-50 bg-black bg-opacity-60" />
+      <Dialog.Overlay className="fixed inset-0 z-50 bg-black bg-opacity-50" />
 
-      <div className="p-8 flex flex-col justify-start z-50 text-white bg-blue rounded-md break-all">
-        <Dialog.Title className="text-xl font-bold">{title}</Dialog.Title>
+      <div className="p-10 w-10/12 sm:5/12 shadow-mainShadow flex flex-col justify-start z-50 text-white bg-black rounded-md break-all">
```
Comment (MatthiasWanner) [inline src/components/Modal.tsx:26]: You've forgot a w "sm:w-5/12"

**Review by MatthiasWanner — CHANGES_REQUESTED** [review]

**Review by MatthiasWanner — APPROVED** [review]

**Pull request merged**